server/src/test/FoldingRangesRequestHandler.test.ts

```diff
@@ -116,6 +116,7 @@ Country myCountry // 78 → 85
         const foldingRanges: FoldingRange[] = handler(DEFAULT_REQUEST_PARAMS);
         expect(foldingRanges).toBeTruthy();
         const expectedFoldingRanges = [
+            FoldingRange.create(1, 1), // Class name
```

Comment:
Noob question: Does 1–1 create, like, a visible folding button that does nothing?

Reply:
It does nothing.